cirq-google/cirq_google/engine/engine.py

```diff
@@ -355,6 +360,49 @@ async def run_sweep_async(
             ValueError: If either `run_name` and `device_config_name` are set but
                 `processor_id` is empty.
         """
+
+        if _STREAM_FEATURE_FLAG:
+            print(
+                '\nRunning using the Quantum Engine stream RPC. To revert to unary RPCs, '
```

Comment:
I could see that, although given that I don't expect the feature flag to stay around for long (probably no longer than 3 months), I'm erring towards over-notifying and having the switch readily accessible when the user needs it most. WDYT?

I'll also add a TODO for removing the feature gate.

Reply:
If you must notify, doing this at most once should be sufficient. e.g., this could be done when the engine or context is created, but I would prefer not to notify go general users since the "stream" vs "unary" feature is opaque to them.

Reply:
Chatted offline - will remove the log message.